packages/nextjs/src/browser.ts

```diff
@@ -0,0 +1,10 @@
+import * as SentryBrowser from '@sentry/browser';
+
+import { NextjsOptions } from './options';
+
+/** Inits the Sentry NextJS SDK on the browser. */
+export function init(options: NextjsOptions): any {
+  SentryBrowser.init(options);
+}
+
+export * from '@sentry/minimal';
```

Comment:
If we export `minimal` we can change the browser import

```suggestion
import { init } from '@sentry/browser';

import { NextjsOptions } from './options';

/** Inits the Sentry NextJS SDK on the browser. */
export function init(options: NextjsOptions): any {
 init(options);
}

export * from '@sentry/minimal';
```